docs/content/how-to-develop-a-workflow-definition-in-go.md

```diff
@@ -92,6 +94,7 @@ In Go specifically, Workflow Definition code can not directly do the following:
 
 - Iterate over maps using `range`, because with `range` the order of the map's iteration is randomized (Use a Side Effect or an Activity.
```

Comment:
```suggestion
- Iterate over maps using `range`, because with `range` the order of the map's iteration is randomized. (Use a Side Effect or an Activity instead.)
```